### DISCUSSION TITLE
Flexible floating objects

### DISCUSSION THREAD

**Discussion by MaryamBC** [DualSPHysics/DualSPHysics#232] [DUALSPHYSICS_V5.2]
Hello DualSPHysics users,

I would like to use the Chrono solver to simulate a flexible object sliding on a surface. Can we use a floating flexible object with link-hinge joints, or are link-hinge joints only suitable for objects attached to a fixed point?

Thank you!

**Comment by imestevez** [comment]
Hello,

Yes, you can use the coupling with Chrono to simulate flexible objects with hinges.
Here is a publication where that approach was used: 

**Capasso, S., Tagliafierro, B., Martínez-Estévez, I., Domínguez, J. M., Crespo, A. J. C., & Viccione, G. (2022). A DEM approach for simulating flexible beam elements with the Project Chrono core module in DualSPHysics. Computational Particle Mechanics, 9(5), 969–985. https://doi.org/10.1007/s40571-021-00451-9**

**Comment by scapass0** [comment]
Hello! 

Hinge-linked objects can be used in both configurations, if they're not restrained to fixed elements and if they're just mutually linked. 

Please refer to the above mentioned paper for reference.

Best wishes
